packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -368,6 +368,45 @@ const TOO_MANY_SNAPSHOT_COLUMNS = [
   },
 ]
 
+const GUEST_TOOLS_COLUMNS = [
+  {
+    name: _('vmNameLabel'),
+    itemRenderer: vm => <Link to={`vms/${vm.id}`}>{vm.name_label}</Link>,
+    sortCriteria: vm => vm.name_label,
+  },
+  {
+    name: _('vmNameDescription'),
+    itemRenderer: vm => vm.name_description,
+    sortCriteria: vm => vm.name_description,
+  },
+  {
+    name: _('vmContainer'),
+    itemRenderer: vm => <VmColContainer id={vm.$container} />,
+  },
+  {
+    default: true,
+    name: _('guestToolStatusColumn'),
+    itemRenderer: vm => {
+      if (!vm.pvDriversDetected) {
+        return _('noToolsDetected')
+      }
+      if (!vm.managementAgentDetected) {
+        return _('managementAgentNotDetected')
+      }
+
+      const version =
+        get(() => vm.pvDriversVersion.split('.')[0]) > 0
```

Comment:
Just noticed that this `get()` (`lodash`) is not the same as [`get`](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/xo-app/vm/tab-general.js#L62) imported here [xo-app/vm/tab-general.js](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/xo-app/vm/tab-general.js#L3). Any suggestions for how to solve this in a nice way?

Reply:
Should I rename the import or change the [use of the `lodash get`](https://github.com/vatesfr/xen-orchestra/blob/303db852ecd6c34e90bb695c23bbad0f1986f543/packages/xo-web/src/xo-app/dashboard/health/index.js#L555) to use the same one as in xo-app/vm/tab-general.js (am I understanding correct if I say that they both functions solve similar problems)?

Reply:
You guessed right :)